### DISCUSSION TITLE
Tailwindcss @import "tailwindcss/utilities" causes Google recaptCHA to fail help

### DISCUSSION THREAD

**Discussion by w6612821** [tailwindlabs/tailwindcss#5564] [Help]
What is the problem with using tailwindcss in vite that causes Google recaptCHAv2 to fail when clicked

**Comment by simonswiss** [comment]
Hey!

Do you have a concrete example of a scenario where this happens, in a code example repro?

**Reply by w6612821** [reply]
in index.css
![image](https://user-images.githubusercontent.com/56861299/134382304-f6bd64c1-1a6c-46a2-b6ad-1cb70218f96d.png)
Sorry for not being able to show you the content in the component. The specific situation is that after the introduction of google recaptcha, google recaptcha can be displayed on the page. When I click the verification, it will keep turning and cause the page to crash, but when I remove @tailwind utilities and restart Then it will return to normal.I use vite for vue3
